src/Umbraco.Web.UI.Client/src/views/propertyeditors/listview/layouts.prevalues.html

```diff
@@ -29,7 +29,7 @@
          </div>
 
          <div>
-            <input ng-if="layout.isSystem === 1" type="checkbox" ng-model="layout.selected" />
+             <umb-checkbox model="layout.selected" class="mce-cmd" />
```

Comment:
@poornimanayar I'm missing the `ng-if="layout.isSystem === 1"` check here. We should preserve this.

Reply:
@poornimanayar you need to update this to the following: `<umb-checkbox ng-if="layout.isSystem === 1" model="layout.selected" />`

Reply:
Thanks @bjarnef for reminding me about this PR. I had totally forgotten about this. I will look at this once I get back from the PL fest. Added to my to-do list :-)

Reply:
That would be great :)
I also noticed the listview layout delete/trash icon in custom listview layouts can't be focused using tab key.

Changing the following to a button element seems to fix this.

```
<i class="icon-trash" ng-click="vm.showPrompt(layout)"></i>
```

```
<button type="button" class="btn-link" ng-click="vm.showPrompt(layout)">
    <i class="icon-trash" aria-hidden="true"></i>
</button>
```

I guess this is a great task in another PR for Mr. @BatJan 😁👍🦇

Reply:
Uh, love the bat emoji! 😄 - Once this PR is done I will have a look at doing some fallback texts if that's what you have in mind @bjarnef ? 😃

Reply:
@BatJan yes, that emoji is very cool! 🦄 
It was not much related to this PR, but just the listview layout delete buttons - only shown for custom layouts.

So this line:

```
<i class="icon-trash" ng-click="vm.showPrompt(layout)"></i>
```

needs to be changed to something like this (an anchor might also work, but I guess a `button` is ideal:

```
<button type="button" class="btn-link" ng-click="vm.showPrompt(layout)">
    <i class="icon-trash" aria-hidden="true"></i>
</button>
```

Reply:
@bjarnef Aaah, got you! 😉